docs/community/uTVLFAQ.md

```diff
@@ -28,12 +28,6 @@ Maybe! People active in the UMA ecosystem and other DeFi communities are. The ea
 2. Then search for that address in this [list](https://github.com/UMAprotocol/token-distribution/blob/master/outputs/governance_recipients.json). 
```

Comment:
Should this entire page be removed. I know it is not what this PR is about but worth asking

Reply:
@mhairimca has another pr to update this to reflect the expiry. I think that is good without removing it, since people might still have questions about it especially when trying to settle.